apps/src/music/blockly/blockUtils.js

```diff
@@ -56,3 +68,78 @@ export const isBlockInsideWhenRun = block => {
     block.getRootBlock().type
   );
 };
+
+// Override default function block implementation for the current block mode.
+export function installFunctionBlocks(blockMode) {
+  if (blockMode === BlockMode.ADVANCED) {
+    Blockly.cdoUtils.registerCustomAdvancedProcedureBlocks();
+    // Re-define blocks from core, in case they were deleted for Simple2 mode.
+    restoreBlockDefinitions();
+    // Replaces "variable:" with "parameter:" block text for added parameters
+    Blockly.Msg['PROCEDURE_VARIABLE'] = musicI18n.parameterLabel();
+  } else {
+    Blockly.cdoUtils.registerCustomProcedureBlocks(true);
```

Comment:
Is the `true` needed here? looks like `registerCustomProcedureBlocks` doesn't take any params right?

Reply:
Not needed! I experimented with adding a param at one point and missed cleaning this up. Thanks!